google/cloud/spanner/client_test.cc

```diff
@@ -981,6 +981,58 @@ TEST(ClientTest, CommitStats) {
   EXPECT_EQ(42, result->commit_stats->mutation_count);
 }
 
+TEST(ClientTest, MaxCommitDelayUnset) {
+  auto timestamp =
+      spanner_internal::TimestampFromRFC3339("2020-10-20T02:20:09.123Z");
+  ASSERT_STATUS_OK(timestamp);
+  CommitStats stats{42};
+
+  auto conn = std::make_shared<MockConnection>();
+  EXPECT_CALL(*conn, Commit)
+      .WillOnce([&timestamp, &stats](Connection::CommitParams const& cp) {
+        EXPECT_TRUE(cp.options.return_stats());
+        EXPECT_FALSE(
+            static_cast<Options>(cp.options).has<MaxCommitDelayOption>());
+        return CommitResult{*timestamp, stats};
+      });
+
+  Client client(conn);
+  auto result =
+      client.Commit(Mutations{}, Options{}.set<CommitReturnStatsOption>(true));
+  ASSERT_STATUS_OK(result);
+  EXPECT_EQ(*timestamp, result->commit_timestamp);
+  ASSERT_TRUE(result->commit_stats.has_value());
+  EXPECT_EQ(42, result->commit_stats->mutation_count);
+}
+
+TEST(ClientTest, MaxCommitDelaySet) {
+  auto timestamp =
+      spanner_internal::TimestampFromRFC3339("2020-10-20T02:20:09.123Z");
+  ASSERT_STATUS_OK(timestamp);
+  CommitStats stats{42};
+
+  auto conn = std::make_shared<MockConnection>();
+  EXPECT_CALL(*conn, Commit)
+      .WillOnce([&timestamp, &stats](Connection::CommitParams const& cp) {
+        EXPECT_TRUE(cp.options.return_stats());
+        EXPECT_TRUE(
+            static_cast<Options>(cp.options).has<MaxCommitDelayOption>());
+        EXPECT_EQ(static_cast<Options>(cp.options).get<MaxCommitDelayOption>(),
+                  std::chrono::milliseconds(100));
+        return CommitResult{*timestamp, stats};
+      });
+
+  Client client(conn);
+  Options options;
+  options.set<CommitReturnStatsOption>(true);
```

Comment:
Please remove the setting/testing of `commit_stats` from the `max_commit_delay` test.

Reply:
Done.